kitsune/questions/templates/questions/new_question_react.html

```diff
@@ -7,7 +7,7 @@
 {% block breadcrumbs %}{% endblock %}
 
 {% set hide_aaq_link = True %}
-{% set hide_signout_link = True %}
+{% set hide_signin = True %}
```

Comment:
nit: i think we should name this variable something else since it hides the sign in, sign out and contributor tools.